.eslintrc.json

```diff
@@ -74,10 +78,12 @@
       },
       "extends": [
         "plugin:@typescript-eslint/recommended",
-        "plugin:@typescript-eslint/recommended-requiring-type-checking"
+        "plugin:@typescript-eslint/recommended-requiring-type-checking",
+        "plugin:import/typescript"
       ],
       "plugins": ["@typescript-eslint"],
       "rules": {
+        "import/no-unresolved": "off",
```

Comment:
LGTM, just wondering why do we need this?
```suggestion
        "import/no-unresolved": "off",
```

Reply:
It's to avoid getting two errors for a single mistake. TypeScript already tells that the module cannot be resolved, so we don't need Eslint doing the same.